Tests/AppTests/Controllers/Registration/CancelRegistrationTest.swift

```diff
@@ -90,40 +74,21 @@ class CancelRegistrationTest: TestCase {
       return
     }
     
-    let headers: [HeaderKey: String] = [
-      TestConstants.Header.Key.userToken: userAnswers.sessionToken,
-      TestConstants.Header.Key.contentType: TestConstants.Header.Value.applicationJson
-    ]
-    
-    let res = try drop
-      .userAuthorizedTestResponse(
-        to: .post,
-        at: "event/register",
-        headers: headers,
-        body: userAnswers.body)
+    let response = try eventRegistration(userAnswers.body, token: userAnswers.sessionToken)
       .assertStatus(is: .ok)
     
-    guard let eventRegId = try res.json?.get("id") as Int! else {
+    guard let eventRegId = try response.json?.get("id") as Int! else {
       XCTFail("Can't get eventRegId from 'event/register' response")
       return
     }
     
     let wrongToken = try getWrongToken(rightToken: userAnswers.sessionToken)!
-    
-    let headersWithWrongToken: [HeaderKey: String] = [
-      TestConstants.Header.Key.userToken: wrongToken,
-      TestConstants.Header.Key.contentType: TestConstants.Header.Value.applicationJson
-    ]
-    
-    let delres = try drop
-      .userAuthorizedTestResponse(
-        to: .delete,
-        at: "event/register/\(eventRegId)",
-        headers: headersWithWrongToken)
+
+    let deleteResponse = try cancellationOfRegistration(eventRegId, token: wrongToken)
```

Comment:
we usually call function  `doSomething`, so `cancelRegistration(:, token:)` seems better, no?

Reply:
sure, fixed!